tsconfig.json

```diff
@@ -12,26 +12,21 @@
  *   Broadcom, Inc. - initial API and implementation
  */
 
- {
-    "compilerOptions": {
-        "module": "commonjs",
-        "target": "es6",
-        "outDir": "out",
-        "lib": [
-            "es7"
-        ],
-        "skipLibCheck": true,
-        "sourceMap": true,
-        "strict": true, /* enable all strict type-checking options */
-        /* Additional Checks */
-        "noImplicitAny": false
-        // "noImplicitReturns": true, /* Report error when not all code paths in function return a value. */
-        // "noFallthroughCasesInSwitch": true, /* Report errors for fallthrough cases in switch statement. */
-        // "noUnusedParameters": true,  /* Report errors on unused parameters. */
-    },
-    "exclude": [
-        "node_modules",
-        ".vscode-test"
-    ],
-    "include": ["src", "test", "__mocks__"]
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "target": "es6",
+    "outDir": "out",
+    "lib": ["es7"],
+    "skipLibCheck": true,
+    "sourceMap": true,
+    "strict": true /* enable all strict type-checking options */,
+    /* Additional Checks */
+    "noImplicitAny": false
```

Comment:
We should enable this ASAP.

Reply:
I agree. + others. Thanks for bringing it up.